### DISCUSSION TITLE
Issue regarding training data

### DISCUSSION THREAD

**Discussion by Talonkarrde89** [ho-dev/HattrickOrganizer#893] [Q&A]
Hello,

Yesterday I had a message "db corrupted, please migrate to new version" with my old HO 1.43. I discovered this new version on Github and I installed it. And when I say installed it, I use the portal version.

I imported my old databases, First I selected the newest db. After I retried it slecting all my db since 2019...

But nothing change, the data regardeing the training of my player is not rebuilt, meaning my player are stuck at "11" value and not the decimal "11.23" ...

I was like, ok, perhaps with the next training session the tool will reset and start again to cumulate decimal. But it's not working. Yesterday I download the last training session values, the system detected one skill change but the other player did not get decimal incrementation...

Do you have an idea on how to fix this ?

Thanks !

(by the way, thanks for the free time spent on this good app :) )

**Comment by wsbrenk** [comment]
concerning missing training effect, we have an issue with poor man's cup matches, which are not loaded when calculating training effect. we have a fix in the latest dev version. this one is for testing purpose only. a corrupt database cannot be excluded when it is used.

database corruption most likely happens if you kill a hattrick session (task manager of pc shutdown). Then you use one of your previous database backups, but not the latest one, which is the corrupted one.

**Reply by Talonkarrde89** [reply]
Hi.

Concerning the training, not even the league had been incremented on player who played them. Is it what your are talking about?

Thanks

**Reply by Talonkarrde89** [reply]
Hi, I just tried the function File>Subskill recalculation (7 weeks) and it has shown decimals on my players. I will see next friday if the incrementation is working well.
